kolibri/core/assets/src/api-resource.js

```diff
@@ -324,9 +344,16 @@ class Collection {
     return this.models.reduce((synced, model) => synced && model.synced, this._synced);
   }
 
+  /**
+   * Set this Collection as synced or not, for true, will also set all models cached in it
+   * as synced.
+   * @param  {Boolean} value Is this Collection synced or not?
+   */
   set synced(value) {
     this._synced = value;
```

Comment:
Realized that if we set our collection synced to false, it would set all the models in it to true! Not quite ideal!

Thought the best way to deal with this for setting false was to only change the Collection, but for true, I have to set them all to true, as the Collection's `synced` getter does an AND on `_synced` and all the Model's `synced` attributes.